deploy.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+sudo apt-get install -qq sshpass
```

Comment:
It would be better to do this in the install phase if possible. 
This way it can be cached.